Digital Twin/procedures/collab.indoorTracking.vail

```diff
@@ -0,0 +1,27 @@
+PROCEDURE collab.indoorTracking(event Object, results Object)
+
+event = event.newValue
+
+var retVal = []
+for result in results{
+    
+    // are we on the same floor
+    if (result.floorid == event.floorid){
+        // formula: distance = square root of (x2 -x1)squared + (y2 - y1)squared
+        var rx = result.fplocation.coordinates[1]
+        var ry = result.fplocation.coordinates[0]
+    	var distance = Math.sqrt( Math.pow((rx - event.x),2) + Math.pow((ry - event.y),2) )
+		result.distance = distance
+
+		push(retVal, result)
+
+		
+
+		//log.info(stringify(distance))
+
+    }
+}
+
+log.info(stringify(retVal))
```

Comment:
log statements in VAIL should be reduced to debug level so users can selectively enable them, but they are not enabled by default.